triton-client/src/components/DatasetCard.tsx

```diff
@@ -52,20 +75,27 @@ function DatasetCard({ datasetID }: DatasetCardProps) {
 			const req = requestByType[type]
 			if (req) {
 				const { type, status, expiry_date } = req
-				return <Button key={type} style={{ paddingLeft: '4', paddingRight: '4' }}>
-					{type}
-					<Divider type={'vertical'} style={{ backgroundColor: 'black' }}/>
-					{status === 'SUCCESS' ? expiry_date ?? '?' : status}
+				return <Button key={type} style={{ paddingLeft: '4', paddingRight: '4' }} disabled={updatingRequest}>
+					<Space>
+						{type}
+						{"|"}
+						{status === 'SUCCESS'
+							? [status, "|", `Expires: ${expiry_date ? expiry_date : "???"}`]
```

Comment:
If  there is no expiry date just write "Never" instead of "???". It is clearly not the desired result but it is clearer to everyone what that mean.

Reply:
The assumption is that we're supposed to have `expiry_date`. If it's still undefined, it's most likely either still loading or mistakenly left out.